clang/lib/Format/Format.cpp

```diff
@@ -3850,6 +3850,7 @@ LangOptions getFormattingLangOpts(const FormatStyle &Style) {
   // the sequence "<::" will be unconditionally treated as "[:".
   // Cf. Lexer::LexTokenInternal.
   LangOpts.Digraphs = LexingStd >= FormatStyle::LS_Cpp11;
+  LangOpts.RawStringLiterals = LexingStd >= FormatStyle::LS_Cpp11;
```

Comment:
As clang-format formats all C code as C++, there's no need to set the new `RawStringLiterals` option here.

Reply:
A bunch of the clang-format tests were failing before I made this change, but I think that may have been before I made some more changes to this; I’ll take another look at this.

Reply:
Can we set the default to 1 in `LangOpts.def`? That would take care of it.

Reply:
That sounds like it might work because it’s overridden by the standard anyway; I’ll try that.

Reply:
That seems to have worked

Reply:
So can undo the change to Format.cpp?

Reply:
I think I’ve already done that; I just haven’t commited that yet iirc.